frontend/src/api/k8s/__tests__/templates.spec.ts

```diff
@@ -59,32 +59,35 @@ const createServingRuntime = (name: string): K8sDSGResource => ({
 describe('assembleServingRuntimeTemplate', () => {
   it('should assemble serving runtime template with templateName', () => {
     const servingRuntimeMock = JSON.stringify(createServingRuntime('template-1'));
+    const servingRuntimeTemplatesMock = mockServingRuntimeTemplateK8sResource({
+      platforms: [ServingRuntimePlatform.MULTI],
+    });
+    delete servingRuntimeTemplatesMock.metadata.ownerReferences;
```

Comment:
these `delete`s shouldn't be necessary right? having an empty `ownerReference` should return `false` from `ownedByDSC`

Reply:
I assume this is here because `assembleServingRuntimeTemplate` won't include an ownerReferences array which makes `toStrictEqual` fail below. You could also avoid this by just changing `mockServingRuntimeTemplateK8sResource` so that it doesn't have a default value of `[]` for `ownerReferences` - if they aren't passed, let them be undefined on the object